### ISSUE TITLE
Deltoid tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11004]
**Preferred term label:**
Deltoid tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Tear in the deltoid muscle or tendon

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Deltoid muscle rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001457
- label: Deltoid muscle tear
- parents: [Abnormality of the shoulder girdle musculature (HP:0001435)](https://hpo.jax.org/app/browse/term/HP:0001435)
- pmid: 
- def: An injury of the deltoid muscle of the shoulder or its tendon characterized by partial or complete rupture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11004
- synonyms:
